data/planet/cwn/ocaml-weekly-news-23-jul-2024.md

```diff
@@ -0,0 +1,12 @@
+---
+title: OCaml Weekly News, 23 Jul 2024
+description:
+url: https://alan.petitepomme.net/cwn/2024.07.23.html
+date: 2024-07-23T12:00:00-00:00
+preview_image:
+authors:
+- Caml Weekly News
+source:
+---
+
+<ol><li><a href="https://alan.petitepomme.net/cwn/2024.07.23.html#1">A Tour of the Living Library -- A Safer FFI</a></li><li><a href="https://alan.petitepomme.net/cwn/2024.07.23.html#2">First Release of `rpmfile`</a></li><li><a href="https://alan.petitepomme.net/cwn/2024.07.23.html#3">Dune Dev Meeting</a></li><li><a href="https://alan.petitepomme.net/cwn/2024.07.23.html#4">Fighting Mutation With Mutation in Living</a></li><li><a href="https://alan.petitepomme.net/cwn/2024.07.23.html#5">A Small Extension of Bigarray.Genarray Adding Iteration, Mapping, and Folding</a></li><li><a href="https://alan.petitepomme.net/cwn/2024.07.23.html#6">cudajit: Bindings to the ~cuda~ and ~nvrtc~ libraries</a></li><li><a href="https://alan.petitepomme.net/cwn/2024.07.23.html#7">Rpmfile 0.2.0 - Changelog</a></li><li><a href="https://alan.petitepomme.net/cwn/2024.07.23.html#8">Exploring the Docusaurus+`odoc` Combo</a></li><li><a href="https://alan.petitepomme.net/cwn/2024.07.23.html#9">Mopsa 1.0 -- Modular Open Platform for Static Analysis</a></li><li><a href="https://alan.petitepomme.net/cwn/2024.07.23.html#10">OCaml 5 Performance</a></li><li><a href="https://alan.petitepomme.net/cwn/2024.07.23.html#11">Other OCaml News</a></li></ol>
```

Comment:
for "~cuda~ and ~nvrtc~" -- should those be subscript, strikethrough, or monospace `cuda` and `nvrtc`?